tasks/main.yml

```diff
@@ -10,3 +10,10 @@
   copy:
     dest: /etc/security/namespace.d/tmp.conf
     src: tmp.conf
+
+- name: Set boolean for SELinux system
+  seboolean:
+    name: polyinstantiation_enabled
+    state: yes
+    persistent: yes
+  when: ansible_selinux is not False
```

Comment:
I guess so. At first I tried "when: ansible_selinux is defined", but it seems to be defined to false even on platform without selinux (and to "a structure" when ansible_selinux is set). I think I did see a discussion also on changing that, so I should maybe dig more this part.